packages/basemap/pyproject.toml

```diff
@@ -1,15 +1,38 @@
 [build-system]
 requires = [
-    'setuptools',
+    'setuptools>=42',
     'wheel',
-    'numpy == 1.26.1; python_version >= "3.12"',
-    'numpy == 1.23.3; python_version == "3.11"',
-    'numpy == 1.21.4; python_version == "3.10"',
-    'numpy == 1.21.4; sys_platform == "darwin" and (python_version >= "3.7" and python_version <= "3.9")',
-    'numpy == 1.16.6; sys_platform != "darwin" and (python_version >= "3.7" and python_version <= "3.9")',
-    'numpy == 1.16.6; python_version == "2.7" or (python_version >= "3.4" and python_version <= "3.6")',
-    'numpy == 1.11.3; python_version == "2.6" or (python_version >= "3.2" and python_version <= "3.3")',
-    'cython >= 0.29, < 3.1; python_version >= "3.3" or python_version < "3.0"',
-    'cython >= 0.26, < 0.27; python_version == "3.2"'
+    'numpy>=1.21.4; python_version == "3.9"',
```

Comment:
The build requirements pin `numpy` with `==` and not with `>=` because we want to use the oldest numpy version that works with a given Python version to build the wheels. This guarantees that, when we run the built wheels with newer `numpy` versions, they still work (the opposite is not guaranteed).

On the other hand, building with `numpy<2.0` will not work with `numpy>=2.0`, so Python versions intended to run with `numpy>=2.0` need to be built with a fixed `numpy` version that is at least 2.0. We later need to see if the wheels built with `numpy>=2.0` still work with `numpy<2.0`, I think they tried to make it in this manner but I cannot confirm.

Build requirements and runtime requirements are different, i.e. `numpy` needs `==` when building, but needs `>=` as runtime requirement.

Reply:
Thanks for clarifying!